### DISCUSSION TITLE
We should avoid "standard" switch name for msbuild driver

### DISCUSSION THREAD

**Discussion by MarcoRossignoli** [coverlet-coverage/coverlet#1252] [Ideas]
Usually I work with a VS command prompt.
Doing some test I found that "strange" include filter are present
```
C:\Program Files (x86)\Microsoft Visual Studio\2019\Preview\VC\Tools\MSVC\14.24.28117\ATLMFC\include;C:\Program Files (x86)\Microsoft Visual Studio\2019\Preview\VC\Tools\MSVC\14.24.28117\include;C:\Program Files (x86)\Windows Kits\NETFXSDK\4.8\include\um;C:\Program Files (x86)\Windows Kits\10\include\10.0.17763.0\ucrt;C:\Program Files (x86)\Windows Kits\10\include\10.0.17763.0\shared;C:\Program Files (x86)\Windows Kits\10\include\10.0.17763.0\um;C:\Program Files (x86)\Windows Kits\10\include\10.0.17763.0\winrt;C:\Program Files (x86)\Windows Kits\10\include\10.0.17763.0\cppwinrt
```
I've opened an issue on cli repo https://github.com/dotnet/cli/issues/12821
At the end the issue is the name switch `Include` that is used by vs command line env and used by msbuild.

I think that we should avoid to have name that collide with "standard" switch name, for instance we should prepend every switch with "Coverlet" i.e.
```
λ dotnet test --filter "TestCategory!=FailureExpected" -v:m /p:CoverletCollectCoverage=true /p:CoverletInclude=....
```

It's a breaking change.

cc: @tonerdo @petli

**Comment by tonerdo** [comment]
I agree. But we can't make this change without breaking existing users. We could alias existing options though, so users can use the "Coverlet" prepended ones when they run into name clash issues

**Comment by Bertk** [comment]
Closing this discussion since it’s no longer relevant to the current state of the project.
